src/lib/Components/Inbox/Conversations/ZeroStateInbox.tsx

```diff
@@ -53,21 +60,23 @@ export default () =>
   <CenteredView>
     <VerticalLayout>
       <Title>Buying Art on Artsy is Simple </Title>
-      <HorizontalLayout>
-        <Icon source={require("../../../../../images/find.png")} />
-        <SmallHeadline>Follow artists and find works you love</SmallHeadline>
-      </HorizontalLayout>
-      <HorizontalLayout>
-        <Icon source={require("../../../../../images/contact.png")} />
-        <SmallHeadline>Contact galleries or bid in auctions to purchase the work</SmallHeadline>
-      </HorizontalLayout>
-      <HorizontalLayout>
-        <Icon source={require("../../../../../images/message.png")} />
-        <SmallHeadline>Find your ongoing conversations and bidding activity here</SmallHeadline>
-      </HorizontalLayout>
-      <HorizontalLayout>
-        <Icon source={require("../../../../../images/pay.png")} />
-        <SmallHeadline>Easily process payment through our secure platform</SmallHeadline>
-      </HorizontalLayout>
+      <List>
```

Comment:
Anyone know if I can make this element conditional? if I remove the `<List>` and `</List>` further down, iPhone works perfectly, but iPad breaks. And vice versa. Even if I empty the `List` styled component, iPhone still breaks. It just doesn't like the extra wrapper view there. 

cc @alloy @orta

Reply:
I don’t really know the answer to that (maybe track down the original author of that code and ask them), but I do think it _should_ be possible to have a wrapper view that doesn’t really do anything.

Reply:
That person is me 😄 

And FYI, I just added the `List`, in order to support iPad (it wasn't there before). 

I agree that it shouldn't make a difference, but if I remove all styling (not even commenting out), it does change. Would love to debug, might take a few mins of our 1-1 for that.

Reply:
I guess it might have to do with default styling that it has? i.e. maybe you can’t leave it empty but do have to override some styling